diff --git a/node_modules/@aws-cdk/aws-eks/lib/cluster-resource.js b/node_modules/@aws-cdk/aws-eks/lib/cluster-resource.js
index f125394..c2062b7 100644
--- a/node_modules/@aws-cdk/aws-eks/lib/cluster-resource.js
+++ b/node_modules/@aws-cdk/aws-eks/lib/cluster-resource.js
@@ -99,6 +99,7 @@ class ClusterResource extends core_2.Construct {
         });
         creationRole.addToPolicy(new iam.PolicyStatement({
             actions: [
+                'ec2:DescribeVpcs',
                 'ec2:DescribeSubnets',
                 'ec2:DescribeRouteTables',
             ],
@@ -134,10 +135,10 @@ class ClusterResource extends core_2.Construct {
         creationRole.addToPolicy(new iam.PolicyStatement({
             actions: ['ec2:DescribeVpcs'],
             resources: [stack.formatArn({
-                    service: 'ec2',
-                    resource: 'vpc',
-                    resourceName: props.vpc.vpcId,
-                })],
+                service: 'ec2',
+                resource: 'vpc',
+                resourceName: props.vpc.vpcId,
+            })],
         }));
         // grant cluster creation role sufficient permission to access the specified key
         // see https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html
